### PR TITLE
#2917

### PR DESCRIPTION
**Pull request by rmakulov**
Severity, minSeverity and severities tests

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/message/MessageMethod.java [hunk 1]
```diff
@@ -24,6 +26,26 @@
...
+    static Map<String, String> aliases;
+
+    static {
+        severities = new HashMap();
+        severities.put("UNDEFINED", 0);
+        severities.put("UNKNOWN", 1);
+        severities.put("NORMAL", 2);
+        severities.put("WARNING", 3);
+        severities.put("MINOR", 4);
+        severities.put("MAJOR", 5);
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/message/MessageMethod.java]: Maybe better to  use Enums?

File: src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        message.setDate(MIN_STORABLE_DATE);
+        for (String severity : stringSeverities) {
+            message.setSeverity(severity);
+            insertMessageCheck(message);
+            message.setDate(Util.addOneMS(message.getDate()));
+        }
+    }
+
+    @BeforeMethod
+    public void prepareQuery() {
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: What is purpose of this method?

Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: Use new query for each method, specify necessary severity, minSeverity or severities inside method.

File: src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                Assert.assertTrue(actualSeverity >= minimumSeverity, "Received severity (" + actualSeverity + ") should be greater than minSeverity (" + minimumSeverity + ")");
+            }
+        }
+    }
+    /*
+         #2917
+         severity is case insensitive
+    */
+    @Test
+    public void testSeverityCaseInsensitive() throws Exception {
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: Maybe parametrize that?

File: src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+            Assert.assertTrue(!severity.matches("-?\\d+(\\.\\d+)?"), "Received severity (" + severity + ") should not be numeric");
+        }
+    }
+    /*
+         #2917
+         minSeverity is >= filter
+    */
+    @Test
+    public void testMinSeverityFilter() throws Exception {
+//        for (Map.Entry<String, Integer> entry : MessageMethod.severities.entrySet()) {
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: What is a purpose of the comment?

File: src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+         #2917
+         response contains severity as name (text) not as numeric code
+    */
+    @Test
+    public void testResponseSeverityNotNumeric() throws Exception {
+        for (Severity s : Severity.values()) {
+            messageQuery.setSeverity(s.name());
+            List<Message> messages = queryMessage(messageQuery).readEntity(new GenericType<List<Message>>() {
+            });
+            String severity = messages.get(0).getSeverity();
```
Comment (unrealwork) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: Maybe use the same enum for message model?

Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: If you want, ok.

File: src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java (hunk 4 above)
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageSeverityQueryTest.java]: I forgot to remove it, thank you.

**Pull request merged**